modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc

```diff
@@ -268,6 +266,29 @@ testtable
 spark-sql ()> select * from testdb.testtable;
 ----
 
+== Use Iceberg connectors with AWS Glue catalog and AWS Lake Formation
+
+The connector can be used with AWS Glue catalog and AWS Lake Formation service.
```

Comment:
```suggestion
The connector can be used with the AWS Glue Data Catalog and the AWS Lake Formation service.
```

Reply:
active voice

Reply:
You can use the the connector with the AWS Glue Data Catalog and AWS Lake Formation service.